src/solvers/postSolver.ml

```diff
@@ -84,8 +87,18 @@ module Verify: F =
       ignore (Pretty.printf "Fixpoint not reached at %a\nOrigin: %a\n @[Solver computed:\n%a\nSide-effect:\n%a\nDifference: %a\n@]" S.Var.pretty_trace y S.Var.pretty_trace x S.Dom.pretty lhs S.Dom.pretty rhs S.Dom.pretty_diff (rhs, lhs))
 
     let one_side ~vh ~x ~y ~d =
-      let y_lhs = try VH.find vh y with Not_found -> S.Dom.bot () in
-      if not (S.Dom.leq d y_lhs) then
+      (* HACK: incremental accesses insanity! *)
+      (* ignore (Pretty.printf "one_side %s: %a\n" (S.Var.var_id y) S.Dom.pretty d); *)
+      let is_acc = String.starts_with (S.Var.var_id y) "access:" in
+      let y_lhs =
+        if is_acc && not (VH.mem accs y) then (
+          VH.replace accs y ();
+          S.Dom.bot ()
+        )
+        else
+          try VH.find vh y with Not_found -> S.Dom.bot ()
+      in
+      if not is_acc && not (S.Dom.leq d y_lhs) then
```

Comment:
Would we not still need some special logic to account for side-effects from things that are superstable?

Reply:
Possibly. When I later thought about it, I was confused about that too, but it somehow works on all the incremental tests we have.

Maybe this is completely broken after all...